dispatch/static/manager/src/js/index.js

```diff
@@ -26,7 +28,14 @@ const store = createStore(
     loadingBar: loadingBarReducer,
     modal: modalReducer
   }),
-  applyMiddleware(thunk, router, promiseMiddleware(), loadingBarMiddleware())
+  applyMiddleware(
+    thunk,
+    router,
+    promiseMiddleware(),
+    loadingBarMiddleware({
+      promiseTypeSuffixes: ['PENDING', 'FULFILLED', 'REJECTED'],
```

Comment:
right but if the returned an action with the action type 'FETCH_ARTICLE_REJECTED' for example, we don't have a specific case set up to handle it in the articles reducer. My question is whether or not we want a specific case to handle this action.